tools/cli/src/webpack/config.ts

```diff
@@ -112,10 +112,13 @@ export const createConfiguration: (
           ? 'js/[name]-[contenthash:8].js'
           : 'js/[name].js',
       // In some cases webpack will emit files starts with "_" which is reserved in web extension.
-      chunkFilename:
-        buildFlags.mode === 'production'
-          ? 'js/chunk.[name]-[contenthash:8].js'
-          : 'js/chunk.[name].js',
+      chunkFilename: pathData => {
+        return pathData.chunk?.name === 'worker'
+          ? 'worker.js'
```

Comment:
no hash for worker?

Reply:
after several rounds of test, it's hard to attach hash to worker file.